sdk/eventhub/azure-eventhubs/azure/eventhub/aio/_eventprocessor/event_processor.py

```diff
@@ -148,85 +131,71 @@ def _context(self, events):
+                                                                 initial_event_position,
+                                                                 event_received_callback)
 
-            try:
-                if self._partition_initialize_handler:
-                    try:
-                        await self._partition_initialize_handler(partition_context)
-                    except Exception as err:  # pylint:disable=broad-except
-                        log.warning(
-                            "EventProcessor instance %r of eventhub %r partition %r consumer group %r. "
-                            " An error occurred while running initialize(). The exception is %r.",
-                            owner_id, eventhub_name, partition_id, consumer_group_name, err
-                        )
-                while True:
-                    try:
-                        events = await partition_consumer.receive()
-                        if events:
-                            if self._track_last_enqueued_event_properties:
-                                self._last_enqueued_event_properties[partition_id] = \
-                                    partition_consumer.last_enqueued_event_properties
-                            with self._context(events):
-                                await self._event_handler(partition_context, events)
-                    except asyncio.CancelledError:
-                        log.info(
-                            "EventProcessor instance %r of eventhub %r partition %r consumer group %r"
-                            " is cancelled",
-                            owner_id,
-                            eventhub_name,
-                            partition_id,
-                            consumer_group_name
-                        )
-                        raise
-                    except Exception as error:  # pylint:disable=broad-except
-                        await self._process_error(partition_context, error)
-                        break
-                    # Go to finally to stop this partition processor.
-                    # Later an EventProcessor(this one or another one) will pick up this partition again
-            finally:
-                await partition_consumer.close()
-                if self._running is False:
-                    await self._close_partition(partition_context, CloseReason.SHUTDOWN)
-                else:
-                    await self._close_partition(partition_context, CloseReason.OWNERSHIP_LOST)
+            if self._partition_initialize_handler:
+                try:
+                    await self._partition_initialize_handler(partition_context)
+                except Exception as err:  # pylint:disable=broad-except
+                    log.warning(
+                        "EventProcessor instance %r of eventhub %r partition %r consumer group %r. "
+                        " An error occurred while running initialize(). The exception is %r.",
```

Comment:
intend for two spaces after the period on the previous line? Also, agree with @johanste to prefer `.format`

Reply:
Format shouldn't be used for logging statements, especially in high-perf scenarios as it will do the formatting regardless of whether logging is enabled, whereas this syntax will evaluate lazily.

Fixed the space though :)